Formula/bitcoin.rb

```diff
@@ -25,7 +25,7 @@ class Bitcoin < Formula
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
-  depends_on "berkeley-db@5"
+  depends_on "berkeley-db@4" # do not upgrade
```

Comment:
Could we add a link to the documentation here?

Please squash the change into your existing commit and use the commit message
```
bitcoind: revert to berkeley-db@4
```

Thanks!

Reply:
Definitely! Added and squashed, let me know if that looks good.